src/orders/components/OrderShippingMethodEditDialog/OrderShippingMethodEditDialog.tsx

```diff
@@ -94,6 +94,7 @@ const OrderShippingMethodEditDialog: React.FC<OrderShippingMethodEditDialogProps
                 name="shippingMethod"
                 data-test-id="shipping-method-select"
                 value={data.shippingMethod}
+                className="select-full-width"
```

Comment:
suggestion: try to wrap Select into Box with width="100%"

Reply:
It doesn't work. Notice how in css I had to select the first child of label in the Select wrapper. I'm overriding this style here https://github.com/saleor/macaw-ui/commit/18bc4255ca8ea8dac8b423c403ffb4a80b493dd0#diff-c9d1697464b9877d7cb8c5cb7285b799dff934013f4f8de11b712bf4eac0127cR62

Reply:
How could I know what you changed if you didn't provide screenshots, Never mind. IMO this is a problem in Macaw we could solve it like this but this is a hacky way.

Reply:
sure - https://github.com/saleor/macaw-ui/pull/843